src/test/regress/expected/rpt.out

```diff
@@ -1329,6 +1353,150 @@ select c from rep_tab where c in (select distinct d from rand_tab);
+     ->  Gather Motion 1:1  (slice1; segments: 1)
+           Output: (f(i))
+           ->  Seq Scan on rpt.t
+                 Output: f(i)
+ Optimizer: Postgres query optimizer
+ Settings: enable_bitmapscan=off, enable_seqscan=off
+(9 rows)
+
+-- ensure we do not make broadcast motion
+explain (costs off, verbose) SELECT (select f(i) from t group by f(i));
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Result
+   Output: $0
+   InitPlan 1 (returns $0)  (slice2)
+     ->  Gather Motion 1:1  (slice1; segments: 1)
+           Output: (f(i))
+           ->  HashAggregate
+                 Output: (f(i))
+                 Group Key: f(t.i)
+                 ->  Seq Scan on rpt.t
+                       Output: i, f(i)
+ Optimizer: Postgres query optimizer
+ Settings: enable_bitmapscan=off, enable_seqscan=off
+(12 rows)
+
+explain (costs off, verbose) SELECT (select i from t group by i having f(i) > 0);
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Result
+   Output: $0
+   InitPlan 1 (returns $0)  (slice2)
+     ->  Gather Motion 1:1  (slice1; segments: 1)
+           Output: i
+           ->  HashAggregate
+                 Output: i
+                 Group Key: t.i
+                 Filter: (f(t.i) > 0)
+                 ->  Seq Scan on rpt.t
+                       Output: i
+ Optimizer: Postgres query optimizer
+ Settings: enable_bitmapscan=off, enable_seqscan=off
+(13 rows)
+
+-- ensure we make broadcast motion
+explain (costs off, verbose) select * from t1 where a in (select random() from t where i=a group by i);
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Gather Motion 3:1  (slice1; segments: 3)
+   Output: t1.a
+   ->  Seq Scan on rpt.t1
+         Output: t1.a
+         Filter: (SubPlan 1)
+         SubPlan 1  (slice1; segments: 1)
+           ->  GroupAggregate
+                 Output: random(), t.i
+                 Group Key: t.i
+                 ->  Result
+                       Output: t.i
+                       Filter: (t.i = t1.a)
+                       ->  Materialize
+                             Output: t.i, t.i
+                             ->  Seq Scan on rpt.t
+                                   Output: t.i, t.i
+ Optimizer: Postgres query optimizer
+ Settings: enable_bitmapscan=off, enable_seqscan=off
+(18 rows)
+
+explain (costs off, verbose) insert into t2 (a, b) select i, random() from t;
+                     QUERY PLAN                      
+-----------------------------------------------------
+ Insert on rpt.t2
+   ->  Broadcast Motion 1:3  (slice1; segments: 1)
+         Output: t.i, (random())
+         ->  Seq Scan on rpt.t
+               Output: t.i, random()
+ Optimizer: Postgres query optimizer
+ Settings: enable_bitmapscan=off, enable_seqscan=off
+(7 rows)
+
+-- ensure we do not break broadcast motion
```

Comment:
Description doesn't fit to the explain output. I think you better describe difference between these queries instead of pointing to expected motion kind only.

Reply:
I don't see any comment changes here to provide fuller description. You've just replaced `ensure we do not break broadcast motion` to `ensure we do not make broadcast motion` instead.